mtc-wasm/assets/css/input.css

```
@@ -1,223 +1,245 @@
-    background: transparent;
-    border-radius: 0;
-}
+@import "tailwindcss";
+@import "tailwindcss/theme.css" layer(theme);
+@import "tailwindcss/preflight.css" layer(base);
+@import "tailwindcss/utilities.css" layer(utilities);
+
+@plugin "@tailwindcss/typography";
+@plugin "daisyui" {
+/*    themes: emerald --default, dracula --prefersdark;*/
+    root: ":root";
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)